org.scala-ide.sdt.core/plugin.xml

```diff
@@ -1838,6 +1838,42 @@
           The font used for all Scala hovers.
        </description>
     </fontDefinition>
+    <colorDefinition
+          categoryId="org.scalaide.ui.theme"
+          id="org.scalaide.ui.color.interpreterForeground"
+          label="Interpreter foreground color"
+          value="60,0,128">
```

Comment:
I think we need to use `defaultsTo` and use a standard colour in the Eclipse platform. That way, ColorThemese plugin will correctly update this one as well (of course, users can still change this to another color).

From the Eclipse extension point docs:

```
defaultsTo - the id of another color definition that is the default setting for the receiver. When there is no preference for this color the color registry will have the value of defaultsTo set for it in the registry. Only one of defaultsTo, value or colorFactory may be defined.
```